_tuning-your-cluster/availability-and-recovery/remote.md

```diff
@@ -192,6 +193,7 @@ You can have the same repository serve as both the segment repository and transl
 {: .note}
 
 As data is added to the index, it also will be continuously uploaded to remote storage in the form of segment and translog files because of refreshes, flushes, and translog fsyncs to disk. Along with data, other metadata files will be uploaded.
+`buffer_interval` field in settings indicates time interval to buffer translog operations. Instead of uploading small translog files, single translog file is created with all the write operations received during the configured interval. This helps in achieving higher throughput but also increases latency. Default value is 100ms.
```

Comment:
```suggestion
The `buffer_interval` setting specifies the time interval to buffer translog operations. Instead of uploading individual translog files, OpenSearch creates a single translog file with all the write operations received during the configured interval. Bundling translog files leads to higher throughput but also increases latency. The default `buffer_interval` value is 100 ms.
```

Reply:
We need a verb or preposition after "time interval". "the time interval during which translog operations are buffered"?